runtime/strings/bytes.cpp

```diff
@@ -69,9 +71,11 @@ SortInt hook_BYTES_bytes2int(
   return move_int(result);
 }
 
-unsigned long get_ui(mpz_t i) {
+unsigned long get_ui_named(mpz_t i, std::string const &caller) {
   if (!mpz_fits_ulong_p(i)) {
-    KLLVM_HOOK_INVALID_ARGUMENT("Integer overflow");
+    std::string const error_msg
+        = "Integer overflow from " + caller + ": " + intToString(i);
+    KLLVM_HOOK_INVALID_ARGUMENT(error_msg);
```

Comment:
This is still performing string concatenation and constructing an unnecessary `std::string` temporary. See [here](https://github.com/runtimeverification/llvm-backend/blob/571667745aefdcfb4820ba13f3c3f5dcabe12489/runtime/arithmetic/int.cpp#L156-L157) for an example of how to use the variadic formatting macro.

Reply:
Thanks for pointing that out! Just fixed it!